war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+
+<web-app
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        version="3.0"
+        metadata-complete="true"
+        xmlns="http://java.sun.com/xml/ns/javaee"
+        xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
+
+    <display-name>lead-capture</display-name>
+    <absolute-ordering/>
+
+    <listener>
+        <listener-class>org.exoplatform.container.web.PortalContainerConfigOwner</listener-class>
+    </listener>
+
+    <filter>
+        <filter-name>ResourceRequestFilter</filter-name>
+        <filter-class>org.exoplatform.portal.application.ResourceRequestFilter</filter-class>
+    </filter>
+
+    <filter>
+        <filter-name>SetCurrentIdentityFilter</filter-name>
+        <filter-class>org.exoplatform.services.security.web.SetCurrentIdentityFilter</filter-class>
+        <init-param>
+            <param-name>restoreIdentity</param-name>
+            <param-value>true</param-value>
+        </init-param>
+    </filter>
+
+    <filter>
+        <filter-name>ThreadLocalSessionProviderInitializedFilter</filter-name>
+        <filter-class>org.exoplatform.frameworks.jcr.web.ThreadLocalSessionProviderInitializedFilter</filter-class>
+    </filter>
+
+    <filter>
+        <filter-name>PortalCheckInitFilter</filter-name>
+        <filter-class>org.gatein.portal.init.PortalCheckInitFilter</filter-class>
+    </filter>
+
+    <filter-mapping>
+        <filter-name>PortalCheckInitFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
+
+    <filter-mapping>
+        <filter-name>SetCurrentIdentityFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
+
+    <filter-mapping>
+        <filter-name>ThreadLocalSessionProviderInitializedFilter</filter-name>
+        <url-pattern>/*</url-pattern>
```

Comment:
I believe this is not needed for the extension

Reply:
right

Reply:
All filters should be removed except ResourceRequestFilter

Reply:
Okey